src/main/java/com/elytradev/smores/registries/FluidRegistry.java

```diff
@@ -27,9 +27,101 @@
 
 package com.elytradev.smores.registries;
 
+import com.elytradev.smores.Smores;
+import com.elytradev.smores.block.BlockFluidSmores;
+import com.elytradev.smores.materials.EnumAlloy;
+import com.elytradev.smores.materials.EnumMetal;
+import net.minecraft.util.ResourceLocation;
+import net.minecraftforge.fluids.Fluid;
+
+import java.util.ArrayList;
+import java.util.List;
+
 public class FluidRegistry {
```

Comment:
Rename to avoid collision with Forge FluidRegistry. Probably SmoresFluids

Reply:
Was thinking that too, it's been a pain in the ass.